docker-compose.yml

```diff
@@ -22,6 +22,31 @@ services:
       - "8080:8080"
       - "8081:8081"
 
+  db:
+    image: postgres:latest
+    networks:
+      - kafka_network
+    environment:
+      POSTGRES_DB: mydb
```

Comment:
Seems like this environment variable will be used to create a database. Maybe more fitting name would be `zeevisiondb`?